runtime-infra/pn-data-monitoring.yaml

```diff
@@ -173,6 +178,79 @@ Resources:
             Action:
               - "sts:AssumeRole" 
 
+  EC2DataMonitoringServiceRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: pn-data-monitoring-ec2-service-role
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - "ec2.amazonaws.com"
+            Action:
+              - "sts:AssumeRole"
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+
+  DataMonitoringEC2ManagedPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      ManagedPolicyName: pn-data-monitoring-ec2-managed-policy
+      Roles:
+        - !Ref EC2DataMonitoringServiceRole
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          # lettura su tutto il bucket dei log e parquet
+          - Effect: Allow
+            Action:
+              - "s3:Get*"
+              - "s3:List*"
+            Resource: 
+              - !Sub "arn:aws:s3:::${LogsBucketName}"
+              - !Sub "arn:aws:s3:::${LogsBucketName}/*"
+              - !Sub "arn:aws:s3:::${DataMonitoringBucket}"
+              - !Sub "arn:aws:s3:::${DataMonitoringBucket}/*"
+          # Accesso in scrittura al bucket dump dell'account corrente nella cartella parquet
+          - Effect: Allow
+            Action:
+              - "s3:PutObject"
+            Resource: 
+              - !Sub "arn:aws:s3:::${DataMonitoringBucket}/${ParquetBucketPath}/*"       
+          # Accessi in lettura al bucket dump dell'account confinfo nella cartella parquet
+          - Fn::If:
+            - IsCoreAccount
+            - Effect: Allow
+              Action:
+                - "s3:Get*"
+                - "s3:List*"
+              Resource: 
+              - !Sub "arn:aws:s3:::${ConfinfoDynamoDbExportBucketName}/${ParquetBucketPath}/*"       
+            - !Ref AWS::NoValue
+          # Accessi in lettura ai log di pn-delivery
+          - Fn::If:
+            - IsCoreAccount
+            - Effect: Allow
+              Action:
+                - "logs:StartQuery"
+                - "logs:StopQuery"
+                - "logs:GetQueryResults"
+              Resource: 
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

ARN in Resource DataMonitoringEC2ManagedPolicy contains hardcoded Partition in ARN or incorrectly placed Pseudo Parameters